### ISSUE TITLE
Not working with Cookiebot?

### ISSUE THREAD

**Issue by Discostu36** [otsmr/inline-popup-blocker#7]
Doesn't seem to work with Cookiebot, at least not when slide-up function is used as on https://www.voffice.pro

**Comment by Discostu36** [comment]
The banner always has `id="CybotCookiebotDialog"`, should be an easy fix.

**Comment by otsmr** [comment]
That's right, you could simply remove it via a block list. But this is not the goal of the project.

The banner is actually even recognized by the function "removeCookieBanner". The problem is that the element itself has other childelements, which are also position fixed, so the banner will not removed:

https://github.com/otsmr/cookie-banner-blocker/blob/main/content/detect.js#L100

```
        let childrenWithFixed = findElementByCssRule("position", null, e => e === "fixed" || e === "absolute", fixedElement)
        if (childrenWithFixed.length > 0) return;
```

This if-condition was made because, as in #2, whole pages were often removed because position fixed was used in the main-div.

**Comment by Discostu36** [comment]
> That's right, you could simply remove it via a block list. But this is not the goal of the project.

I understand that you don't want to create and maintain a blocklist for websites, but maybe one for the most common cookie banner plug-ins could be considered?

**Comment by otsmr** [comment]
Yes, the thing is that there are already very good addons for this and these also already have "CybotCookiebotDialog" on the list.

I personally use  [uBlock Origin](https://addons.mozilla.org/de/firefox/addon/ublock-origin/) for this. There are already a lot of such cookie banner lists under "Filterlisten" > "Belästigungen".

The goal of this addon is really only to detect "new" banners, or also banners where you are redirected (like at golem.de).

**Comment by Discostu36** [comment]
I understand. I am using uBlock Origin but didn't know it can also be used for cookie banners. Thank you!
